rct229/rulesets/ashrae9012019/section4/section4rule11.py

```diff
@@ -0,0 +1,216 @@
+            rmi_b = context.BASELINE_0
+            rmi_p = context.PROPOSED
+            return {
+                "rmi_b": rmi_b,
+                "rmi_p": rmi_p,
+                "baseline_hvac_sys_type_ids_dict_b": get_baseline_system_types(rmi_b),
+                "dict_hvac_sys_zones_served_p": get_hvac_zone_list_w_area_dict(rmi_p),
+            }
+
+        class ZoneRule(RuleDefinitionBase):
+            def __init__(self):
+                super(Section4Rule11.RuleSetModelInstanceRule.ZoneRule, self,).__init__(
+                    rmrs_used=produce_ruleset_model_instance(
+                        USER=False, BASELINE_0=True, PROPOSED=True
+                    ),
+                    required_fields={
+                        "$": ["terminals"],
+                        "terminal": [
+                            "served_by_heating_ventilating_air_conditioning_system"
+                        ],
+                    },
+                )
+
+            def get_calc_vals(self, context, data=None):
+                zone_b = context.BASELINE_0
+                zone_p = context.PROPOSED
+
+                is_leap_year = data["is_leap_year"]
+                baseline_hvac_sys_type_ids_dict_b = data[
+                    "baseline_hvac_sys_type_ids_dict_b"
+                ]
+                dict_hvac_sys_zones_served_p = data["dict_hvac_sys_zones_served_p"]
+                rmi_b = data["rmi_b"]
+                rmi_p = data["rmi_p"]
+
+                fan_schedule_b = get_aggregated_zone_hvac_fan_operating_schedule(
+                    rmi_b, zone_b["id"], is_leap_year
+                )
+                fan_schedule_p = get_aggregated_zone_hvac_fan_operating_schedule(
+                    rmi_p, zone_p["id"], is_leap_year
+                )
+
+                fan_schedule_hourly_values_b = fan_schedule_b.get("hourly_values")
+                fan_schedule_hourly_values_p = fan_schedule_p.get("hourly_values")
+
+                schedule_match = (
+                    fan_schedule_hourly_values_b == fan_schedule_hourly_values_p
+                )
+
+                proposed_served_by_multizone = False
+                hvac_id_b = zone_b["terminals"][0][
+                    "served_by_heating_ventilating_air_conditioning_system"
+                ]
+                hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[
+                    list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_id_b)
+                ]
+
+                baseline_served_by_multizone = hvac_sys_type_b in APPLICABLE_SYS_TYPES
+
+                list_hvac_systems_p = get_list_hvac_systems_associated_with_zone(
+                    rmi_p, zone_p["id"]
+                )
+                for hvac_id_p in list_hvac_systems_p:
+                    if (
+                        len(
+                            list(
+                                dict_hvac_sys_zones_served_p[hvac_id_p][
+                                    "Zone_List"
```

Comment:
This is not `Zone_List`. Checking the [function ](https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/rulesets/ashrae9012019/ruleset_functions/get_hvac_zone_list_w_area_dict.py#L88-L96), you will find the correct key is `zone_list`.
RDS is not always correct on key and value names, so it is better to reference the functions.